aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -514,4 +516,28 @@ static inline void checkLinalgCompatibleDtype(const std::string& fn_name, Scalar
       out_name, " with dtype ", out_type);
 }
 
+// check if can run with oneDNN bf16 gemm path
+static inline bool checkOneDnnBf16GemmUsable(const TensorList& input_args, const Tensor& result=Tensor()) {
```

Comment:
about the naming, current it is still called 'mkldnn', so as for this patch it is more proper to use 'mkldnn'.
we may discuss whether we need to change all 'mkldnn' to 'onednn' in following patches.

and also no need for default parameter here.
and don't need a TensorList here, this might add up overhead for small sized gemm, you probably to do some benchmarking with vtune to check.